src/python/dxpy/dx_extract_utils/input_validation.py

```diff
@@ -171,6 +174,20 @@ def validate_filter(filter, filter_type):
             if len(filter["allele_id"]) > 100:
                 err_exit(maxitem_message.format("allele_id", 100))
 
+        # Check location field
+        if "location" in keys:
+            # Ensure there are not more than 100 locations
+            if len(filter["location"]) > 100:
+                err_exit(maxitem_message.format("location", 100))
+            for indiv_location in filter["location"]:
```

Comment:
Very, very minor suggestion for lines 182 and 183:

```suggestion
            indiv_loc_keys = [indiv_location.keys() for indiv_location in filter["location"]]
```